### DISCUSSION TITLE
Grid Values as Design Tokens

### DISCUSSION THREAD

**Discussion by JoCa96** [SchwarzIT/onyx#201] [Ideas]
Currently all grid values are manually synced from [Figma](https://www.figma.com/file/YfEUBOHk4J4nYrk04geswG/onyx---Design-System-Library?type=design&node-id=219-5876&mode=dev) to our [grid.scss file ](https://github.com/SchwarzIT/onyx/blob/main/packages/sit-onyx/src/styles/grid.scss).

We could load the BREAKPOINT, COLUMNS, GUTTER and MARGIN values via Design tokens, to increase parity between UX design and implementation.

**Comment by JoCa96** [comment]
@jannick-ux Would that be possible?

**Reply by jannick-ux** [reply]
@JoCa96 
Yes the figma variables (=tokens) can be used for building grids. Every unit of the grid (number of columns, margin, gutter) can be described by tokens. We already have all the units with our "spacing-tokens-set". 
Is this what you ment?

Here are information of what can be supported by tokens in figma
https://help.figma.com/hc/en-us/articles/14506821864087#Types_of_variables
